### PR TITLE
fix(apps/interface): red indicator icon svg name

### PR DESCRIPTION
**Pull request by math-marcellino**
1. Fix the svg props
2. Move it to `components/Icons/RedIndicator`

### CODE REVIEW

File: apps/interface/components/Icons/RedIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { Icon, IconProps } from "@chakra-ui/react";
+
+export const RedIndicatorIcon = (props: IconProps) => {
+    return (
+        <Icon
+            data-testid="IndicatorIcon"
```
Comment (pyk) [inline apps/interface/components/Icons/RedIndicator.tsx:6]: @math-marcellino s/IndicatorIcon/RedIndicatorIcon

I think we missed this on BlueIndicatorIcon ?

https://github.com/risedle/monorepo/blob/61f596725cd208834e08d18e6364367c2282399c/apps/interface/components/Icons/BlueIndicator.tsx#L5-L6

update the BlueIndicator too

**Review by pyk — CHANGES_REQUESTED** [review]

File: apps/interface/components/Icons/RedIndicator.tsx (hunk 1 above)
Comment (math-marcellino) [reply apps/interface/components/Icons/RedIndicator.tsx:6]: Ahh yess I think I we missed it sir, okay I'll fix it real quick sir 👍

Comment (math-marcellino) [reply apps/interface/components/Icons/RedIndicator.tsx]: Fixed sir @pyk 👍

**Review by pyk — APPROVED** [review]
LGTM

**Pull request merged**